services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/custom_activity_pack/index.tsx

```diff
@@ -114,7 +117,7 @@ const CustomActivityPack = ({
     number += savedActivityFilters.length ? 1 : 0
     number += flagFilters.length
 
-    activityClassificationGroupings.forEach((g) => {
+    activityClassificationGroupings(true).forEach((g) => {
```

Comment:
I'm assuming that since this is used for calculating a property, it really always should be `true` rather than being dependent on the state of `showComrpehension`.